<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KTeamPositionControllerWidget</class>
 <widget class="QWidget" name="KTeamPositionControllerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KTeamPositionControllerWidget</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="xTargetPosition">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-99.989999999999995</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>xSoll</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>ySoll</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="yTargetPosition">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-99.989999999999995</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLCDNumber" name="xRobotPosition">
     <property name="numDigits">
      <number>6</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>xIst</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>yIst</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLCDNumber" name="yRobotPosition">
     <property name="numDigits">
      <number>6</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="startButton">
     <property name="text">
      <string>Set target-position</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
